ClosedXML/Excel/Rows/XLRows.cs

```diff
@@ -11,7 +11,15 @@ internal class XLRows : XLStylizedBase, IXLRows, IXLStylized
         private readonly List<XLRow> _rows = new List<XLRow>();
         private readonly XLWorksheet _worksheet;
 
-        public XLRows(XLWorksheet worksheet) : base(XLWorkbook.DefaultStyleValue)
+
+        /// <summary>
+        /// Create a new instance of <see cref="XLRows"/>.
+        /// </summary>
+        /// <param name="worksheet">If worksheet is specified it means that the created instance represents
+        /// all rows on a worksheet so changing its height will affect all rows.</param>
+        /// <param name="defaultStyle">Default style to use when initializing child entries.</param>
+        public XLRows(XLWorksheet worksheet, XLStyleValue defaultStyle)
+            : base(defaultStyle ?? XLWorkbook.DefaultStyleValue)
```

Comment:
Instead of falling back to `XLWorkbook.DefaultStyleValue` here, should we not modify `XLStylizedBase`'s constructor to be
```c#
        protected XLStylizedBase(XLStyleValue styleValue)
        {
            StyleValue = styleValue ?? XLWorkbook.DefaultStyleValue;
        }
```

Reply:
And if so, maybe `defaultStyle` should be optional with default of `null`?

Reply:
Good point.